src/store/appoint.js

```diff
@@ -0,0 +1,45 @@
+import Axios from "axios";
+
+const appointmentsUrl = "http://localhost:3000/appointments";
+const clientsUrl = "http://localhost:3000/clients";
+export default {
+  state: {
+    appoints: [],
+    clients: []
```

Comment:
`clients` cannot be in `appointments` store

Reply:
done